src/lib/dnssd/Types.h

```diff
@@ -197,10 +197,22 @@ struct CommonResolutionData
 struct OperationalNodeData
 {
     PeerId peerId;
-
+    bool hasZeroTTL;
     void Reset() { peerId = PeerId(); }
 };
 
+struct OperationalNodeBrowseData : public OperationalNodeData
+{
+    OperationalNodeBrowseData() { Reset(); };
+    void LogDetail() const
+    {
+        ChipLogDetail(Discovery, "Discovered Operational node:\r\n");
+        ChipLogDetail(Discovery, "\tNode ID: " ChipLogFormatX64 "-" ChipLogFormatX64 "\r\n",
```

Comment:
This is really not the "Node ID"... Maybe "Node instance:"?

But also, this seems to be re-inventing ChipLogFormatPeerId/ChipLogValuePeerId.

Reply:
renamed and used ChipLogFormatPeerId+ChipLogValuePeerId for logging.